docs/administer/rbac.md

```diff
@@ -0,0 +1,265 @@
+
+- `role:admin` - unrestricted access to all resources
+
+This default built-in role definition can be seen in [policy.csv](https://github.com/percona/everest/blob/main/deploy/quickstart-k8s.yaml#L91).
+
+## RBAC resources and privileges
+
+Below is a comprehensive table outlining the permissions granted for various **resources**:
+
+??? info "Permissions for resources"
+
+
+    !!! info "Important"
+        If you have permission for specific namespaces or resources, you can perform read, update, create, or delete actions only on those resources or only within those namespaces. However, if you have permission for **all** the resources or namespaces, you can carry out these actions across all the resources and namespaces.
+
+    </br> </br>
+
+    **Table: Privileges for the various resources in Percona Everest**
+
+    |**Resource**| **Read**| **Create**| **Update**|**Delete**|
+    |------------|---------|-----------|-----------|----------|
+    | Namespaces|You can view namespaces| :x:  |:x: | :x:|
+    | Database engines</br> MySQL, MongoDB, PostreSQL|You view can database engines when you create databases</br>  </br>**Note**: This policy must at least be **read all** so the users can create databases.| :x: |Modify database engines| :x:|
+    | Database clusters|You can view databases| You can create databases|You can modify databases| You can delete databases|
+    | Backups storage and Monitoring endpoints|You can siew backups and monitroing endpoints| You can create backups and monitroing endpoints|You can modify backups and monitroing endpoints| You can delete backups and monitroing endpoints|
+    | Database data (credentials)|View database data (credentials) </br> </br>**Note**: If no policy is defined:</br> * You cannot see the credentials and the connection string.</br> * You also cannot create a database from any backup.| :x: |:x:| :x:|
+
+
+## Policy definition in RBAC
+
+RBAC policies are the rules and guidelines that define how roles, permissions, and users are managed within RBAC. These policies ensure that users have appropriate access to resources based on their roles within Percona Everest.
+
+
+The policy definiton in Percona Everest is:
+
+
+```sh
+p, adminrole:role, <resource>, <action>, <namespace>/<resourcename>
+```
+    
+Where:
+
+ **adminrole:role**: refers to the subject. It can also be name of the user.
+
+ **resource**: could be namespaces, database engines, etc.
+
+ **action**: such read, create, update, delete
+
+**namespace/resourcename**: `*/*` is a wildcard indicating all namespaces and all resources.
+. Alternatively it can be of the format `<namespace>/<resourcename>.`
+
+
+### Super Admin policy
+
+Check out the policy for a **Super Admin** role:
```

Comment:
What's Super Admin? Is it any different from admin? I think we should stick with `admin` unless they have different meanings..

Reply:
Fixed